src/commands/exile_commands.py

```diff
@@ -68,14 +68,20 @@ async def exile(
     )
     async def roulette(interaction: discord.Interaction):
         """Test your luck, fail and be exiled..."""
-        exile_duration_options = [0, 1, 6, 12, 18, 24]
-        rand_choice = choice(exile_duration_options)
+        safety_options = [True, True, True, True, True, False]
+        exile_duration_options = [1, 6, 12, 18, 24]
+        safety_choice = choice(safety_options)
+        duration_choice = choice(exile_duration_options)
 
         async with create_response_context(interaction, False) as response_message:
             async with create_logging_embed(interaction) as logging_embed:
-                if rand_choice != 0:
+                if safety_choice:
+                    response_message.set_string(
+                        f"<@{interaction.user.id}> has tested their luck and lives another day..."
+                    )
+                else:
                     reason = "roulette"
-                    exile_duration = calculate_time_delta(f"{rand_choice}h")
+                    exile_duration = calculate_time_delta(f"{duration_choice}h")
```

Comment:
grr it doesnt let me comment on the other file but we should also add the duration to the embed just like we do for exile

Reply:
@brtran4 I think I know what you're calling out, I've added it but I think it's a little confusing since the embed is going to have the duration now regardless of if the user was actually exiled or not. If I'm misunderstanding the comment LMK and I can fix

Reply:
yeah i couldnt remember if we already fixed it or not but seems like it's in another PR. lgtm